aws/rapids_ec2_mnmg.ipynb

```diff
@@ -120,15 +121,16 @@
    "outputs": [],
    "source": [
     "cluster = EC2Cluster(env_vars=get_aws_credentials(),\n",
-    "                     instance_type=\"p3.8xlarge\", # 4 V100 GPUs\n",
-    "                     docker_image=\"rapidsai/rapidsai:21.06-cuda11.0-runtime-ubuntu18.04-py3.8\",\n",
+    "                     instance_type=\"g4dn.12xlarge\",  # 4 T4 GPUs\n",
+    "                     docker_image=\"rapidsai/rapidsai-core-nightly:22.08-cuda11.5-runtime-ubuntu20.04-py3.9\",\n",
```

Comment:
I don't think we want to be using nightlies in these notebooks. We probably want to be using the latest stable instead.

Reply:
@jacobtomlinson How do we test the notebook with 22.08 then? #157 says we should update the notebook to use 22.08.

Reply:
Absolutely, we should be testing with the nightly to make sure 22.08 works but we need to change the value in the example repo to be the stable 22.08 image (that isn't released yet).

I expect we will do a big merging session on the day the 22.08 images are released.